index.js

```diff
@@ -40,7 +40,8 @@ const zlib         = require('zlib');
  // and may log errors.
  // They need to be tested, so they shouldn't have side effects on load.
  const brotli = brotliCompat();
- const zopfli = zopfliCompat();
+ let useZopfliForGzip = true;
+ let   zopfli = zopfliCompat(useZopfliForGzip);
```

Comment:
Spacing after `let`, and this is a bad design - it will always import `node-zopfli-es` because the user won't have had a chance to change this setting yet.